config/management/src/lib.rs

```diff
@@ -409,81 +405,86 @@ pub mod tests {
-            AccountAddress::random(),
-            "/ip4/0.0.0.0/tcp/6180".parse().unwrap(),
-            "/ip4/0.0.0.0/tcp/6180".parse().unwrap(),
-            local_ns,
-            remote_ns,
-        )
-        .unwrap();
+        let local_txn = helper
+            .validator_config(
+                AccountAddress::random(),
+                "/ip4/0.0.0.0/tcp/6180".parse().unwrap(),
+                "/ip4/0.0.0.0/tcp/6180".parse().unwrap(),
+                local_ns,
+                remote_ns,
+            )
+            .unwrap();
 
+        let remote = helper.storage(remote_ns.into());
         let remote_txn = remote.get(constants::VALIDATOR_CONFIG).unwrap().value;
         let remote_txn = remote_txn.transaction().unwrap();
 
         assert_eq!(local_txn, remote_txn);
     }
 
     #[test]
-    #[ignore]
     fn test_verify() {
+        let helper = StorageHelper::new();
         let namespace = "verify";
 
-        let mut storage = default_storage(namespace.into());
-        storage.reset_and_clear().unwrap();
-
-        let output = verify(namespace).unwrap().split("KeyNotSet").count();
+        let output = helper.verify(namespace).unwrap().split("KeyNotSet").count();
         assert_eq!(output, 10); // 9 KeyNotSet results in 9 splits
 
-        initialize_storage(storage.as_mut());
+        helper.initialize(namespace.into());
 
-        let output = verify(namespace).unwrap().split("KeyNotSet").count();
+        let output = helper.verify(namespace).unwrap().split("KeyNotSet").count();
         assert_eq!(output, 1); // 0 KeyNotSet results in 1 split
     }
 
     #[test]
-    #[ignore]
     fn test_owner_key() {
-        test_key(OWNER_KEY, owner_key);
+        test_key(OWNER_KEY, StorageHelper::owner_key);
     }
 
     #[test]
-    #[ignore]
     fn test_operator_key() {
-        test_key(OPERATOR_KEY, operator_key);
+        test_key(OPERATOR_KEY, StorageHelper::operator_key);
     }
 
-    fn test_key(key_name: &str, op: fn(&str, &str) -> Result<Ed25519PublicKey, Error>) {
+    fn test_key(
+        key_name: &str,
+        op: fn(&StorageHelper, &str, &str) -> Result<Ed25519PublicKey, Error>,
+    ) {
+        let helper = StorageHelper::new();
         let local_ns = format!("local_{}_key", key_name);
         let remote_ns = format!("remote_{}_key", key_name);
 
-        let mut local = default_storage(local_ns.clone());
-        local.reset_and_clear().unwrap();
-        op(&local_ns, &remote_ns).unwrap_err();
+        op(&helper, &local_ns, &remote_ns).unwrap_err();
 
-        initialize_storage(local.as_mut());
+        helper.initialize(local_ns.clone());
+        let local = helper.storage(local_ns.clone());
         let local_key = local.get_public_key(key_name).unwrap().public_key;
 
-        let mut remote = default_storage(remote_ns.clone());
-        remote.reset_and_clear().unwrap();
-
-        let output_key = op(&local_ns, &remote_ns).unwrap();
+        println!("{}", helper.path_string());
```

Comment:
Guessing this is a loose end from debugging?

Reply:
yuck, woops